packages/launchpad/src/Main.vue

```diff
@@ -67,7 +67,8 @@
           v-else-if="currentProject.currentTestingType === 'component' && !currentProject.isCTConfigured"
           :gql="query.data.value"
         />
-        <OpenBrowser v-else />
+        <OpenBrowser v-else-if="currentProject.browsers?.length" />
+        <Spinner v-else />
```

Comment:
nice! Was seeing this issue where the browsers would display when it should be a loading page

Reply:
Yep - we went through and I think fixed a number of the edge cases causing this